src/v/cluster/tx_gateway_frontend.cc

```diff
@@ -30,6 +30,7 @@
 #include <seastar/core/coroutine.hh>
 
 #include <algorithm>
+#include <unistd.h>
```

Comment:
what's this for? chances are good that there is a c++ header that will provide what you need.